[2022-05-05 13:34:49,769] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-04T00:00:00+00:00 [queued]>
[2022-05-05 13:34:49,773] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-04T00:00:00+00:00 [queued]>
[2022-05-05 13:34:49,773] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-05-05 13:34:49,773] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-05-05 13:34:49,773] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-05-05 13:34:49,786] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): also_run_this> on 2022-05-04 00:00:00+00:00
[2022-05-05 13:34:49,787] {standard_task_runner.py:52} INFO - Started process 26092 to run task
[2022-05-05 13:34:49,789] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2022-05-04T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', '/home/udimini/airflow_venv/airflow_venv/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpmqin137p', '--error-file', '/tmp/tmpzfrpj3je']
[2022-05-05 13:34:49,790] {standard_task_runner.py:80} INFO - Job 16: Subtask also_run_this
[2022-05-05 13:34:49,823] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2022-05-04T00:00:00+00:00 [running]> on host DESKTOP-O4M0F04.localdomain
[2022-05-05 13:34:49,857] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-05-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-04T00:00:00+00:00
[2022-05-05 13:34:49,858] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-05 13:34:49,858] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2022-05-04T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-04 00:00:00+00:00: scheduled__2022-05-04T00:00:00+00:00, externally triggered: False>"']
[2022-05-05 13:34:49,861] {subprocess.py:85} INFO - Output:
[2022-05-05 13:34:49,862] {subprocess.py:89} INFO - run_id=scheduled__2022-05-04T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-04 00:00:00+00:00: scheduled__2022-05-04T00:00:00+00:00, externally triggered: False>
[2022-05-05 13:34:49,862] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-05 13:34:49,879] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220504T000000, start_date=20220505T113449, end_date=20220505T113449
[2022-05-05 13:34:49,922] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-05 13:34:49,934] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
